extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java

```diff
@@ -220,10 +220,11 @@ private RegistryImage toRegistryImage(ImageReference imageReference, Optional<St
         CredentialRetrieverFactory credentialRetrieverFactory = CredentialRetrieverFactory.forImage(imageReference,
                 log::info);
         RegistryImage registryImage = RegistryImage.named(imageReference);
-        registryImage.addCredentialRetriever(credentialRetrieverFactory.wellKnownCredentialHelpers());
-        registryImage.addCredentialRetriever(credentialRetrieverFactory.dockerConfig());
         if (username.isPresent() && password.isPresent()) {
             registryImage.addCredential(username.get(), password.get());
+        } else {
```

Comment:
My opinion - but I would think that we shouldn't ever include the docker credentials. It's either the quarkus credentials or nothing.

Reply:
Which is what it does now, no?

Reply:
its an `if`/`else`. If the credentials are present in the quarkus config, use them. Otherwise fall back to the docker config.

At least that's how I understand the code.

Reply:
Ah, no. We won't do that.

We definitely do want to include the standard configuration when the user has not supplied credentials